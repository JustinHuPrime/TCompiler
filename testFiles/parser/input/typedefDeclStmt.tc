module foo;

void bar() {
  typedef char **t;
  t p;
}