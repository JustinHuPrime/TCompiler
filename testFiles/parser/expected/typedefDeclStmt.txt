testFiles/parser/typedefDeclStmt.tc (code):
FILE(1, 1, STAB(ENTRY(bar, FUNCTION(testFiles/parser/typedefDeclStmt.tc, 3, 1, void()))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), FUNDEFN(3, 1, KEYWORDTYPE(3, 1, void), ID(3, 6, bar, REFERENCES(testFiles/parser/typedefDeclStmt.tc, 3, 1)), STAB(), COMPOUNDSTMT(3, 12, STAB(ENTRY(t, TYEPDEF(testFiles/parser/typedefDeclStmt.tc, 4, 3, char **)), ENTRY(p, VARIABLE(testFiles/parser/typedefDeclStmt.tc, 5, 5, t))), TYPEDEFDECL(4, 3, ID(4, 18, t, REFERENCES(testFiles/parser/typedefDeclStmt.tc, 4, 3)), MODIFIEDTYPE(4, 11, POINTER, MODIFIEDTYPE(4, 11, POINTER, KEYWORDTYPE(4, 11, char)))), VARDEFNSTMT(5, 3, ID(5, 3, t, REFERENCES(testFiles/parser/typedefDeclStmt.tc, 4, 3)), ID(5, 5, p, REFERENCES(testFiles/parser/typedefDeclStmt.tc, 5, 5))))))
